tools/travis-setup-db.sh

```diff
@@ -340,3 +343,12 @@ elif [ "$DB" = 'mssql' ]; then
 else
     echo "Skip setting up database"
 fi
+}
+
+# The DB env var may contain single value "mysql"
+# or list of values separated with a space "elasticsearch cassandra"
+# in case of list of values all listed database will be set up (or at least tried)
+
+for db in ${DB}; do
```

Comment:
I would rather use DBS, DB_LIST, DATABASES, BASES, ENGINES instead of DB here.
And DB can still be DB.